# Welcome to the biocommons!

The biocommons is an [autonomous
collective](https://youtu.be/ZtYU87QNjPw?t=60) that builds and shares
open source tools for bioinformatics, mostly written in Python.

This page provides an introduction to packages and communications.


# Packages

## [anyvar](https://github.com/biocommons/anyvar)
[in development] Proof-of-Concept variation translation, validation, and registration service

## [bioutils](https://github.com/biocommons/bioutils)
provides common tools and lookup tables used primarily by the hgvs and uta packages

## [eutils](https://github.com/biocommons/eutils)
simplified searching, fetching, and parsing records from NCBI using their E-utilities interface

## [hgvs](https://github.com/biocommons/hgvs)
Python library to parse, format, validate, normalize, and map sequence variants.

## [SeqRepo](https://github.com/biocommons/biocommons.seqrepo)
non-redundant, compressed, journalled, file-based storage for biological sequences

## [seqreo-rest-service](https://github.com/biocommons/seqrepo-rest-service)
OpenAPI-based REST interface to biological sequences and sequence metadata

## [UTA](https://github.com/biocommons/uta)
Universal Transcript Archive: comprehensive genome-transcript alignments; multiple transcript sources, versions, and alignment methods; available as a docker image



# Community

## Mailing Lists

* [biocommons-announce](https://groups.google.com/g/biocommons-announce)
* [biocommons-devops](https://groups.google.com/g/biocommons-devops)
* [biocommons-devs](https://groups.google.com/g/biocommons-devs)
* [biocommons-users](https://groups.google.com/g/biocommons-users)
* [hgvs-discuss](https://groups.google.com/g/hgvs-discuss)

## Slack

* [slack workspace](https://biocommons.slack.com/)

## Documents

The [Google Drive
folder](https://drive.google.com/drive/folders/0ByWpa1IeblpxfnhjUEYya3Ewbjh0UVBzaWVieUdnTUNNY3UtbGhybDQtdTVhVnZBQlg4Nzg?usp=sharing)
has lots of documents, and instructions as well.

